pkg/identity/service.go

```diff
@@ -152,6 +174,11 @@ func (svc *Service) Certify(ctx context.Context, req *pb.CertifyRequest) (*pb.Ce
 		return nil, status.Error(codes.Unavailable, "cert issuer not ready yet")
 	}
 
+	if err := svc.ensureIssuerStillValid(); err != nil {
+		log.Errorf("Could not process CSR because of CA cert validation failure: %s", err)
```

Comment:
Let's use lower case for the first word in the log, per Go's [code review convention](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).